test/e2e/tests/add-hide-token.spec.js

```diff
@@ -69,7 +86,21 @@ describe('Add existing token using search', function () {
   it('renders the balance for the chosen token', async function () {
     await withFixtures(
       {
-        fixtures: 'imported-account',
+        fixtures: new FixtureBuilder()
+          .withTokenListController({
+            tokenList: {
+              '0x0D8775F648430679A709E98d2b0Cb6250d2887EF': {
+                address: '0x0D8775F648430679A709E98d2b0Cb6250d2887EF',
+                decimals: 18,
+                iconUrl:
+                  'https://static.metaswap.codefi.network/api/v1/tokenIcons/1337/0x0d8775f648430679a709e98d2b0cb6250d2887ef.png',
+                name: 'Basic Attention Token',
+                occurrences: null,
+                symbol: 'BAT',
+              },
+            },
+          })
+          .build(),
```

Comment:
Is this needed? The old fixture didn't have it. I'm guessing it's using the fallback `contract-metadata` token list.

```suggestion
        fixtures: new FixtureBuilder().build(),
```

Reply:
Good catch, this was unnecessary as we are now mocking the token list, I've enabled token detection bringing this test in line with how it use to work